packages/vite-plugin-svelte/src/utils/dependencies.ts

```diff
@@ -0,0 +1,156 @@
+import { log } from './log';
+import path from 'path';
+import fs from 'fs';
+import { createRequire } from 'module';
+
+export function findSvelteDependencies(root: string, cwdFallback = true): SvelteDependency[] {
```

Comment:
should this be called `findRootSvelteDependencies` to differentiate it from `getSvelteDependencies`?